.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Issue
+about: Feature/Need/Problem complete template
+title: ''
+labels: blockchain, needs-research
+assignees: ''
+
+---
+
+## Description
+
+_Explain what's the need, what's the desired behaviour_
+
+## Current Behaviour
```

Comment:
```suggestion
## Current Behavior
```

Behaviour is the British spelling.  Nothing wrong with that, but in the other files we use "behavior" so better to make it consistent.